css/templates/e-commerce/page.css

```diff
@@ -557,3 +557,32 @@ amp-selector [option][selected] {
     margin-left: calc(1/12 * 100%);
   }
 }
+
+
+/* CART */
+.commerce-cart-icon {
+  color: var(--color-font-accent);
+}
+
+.commerce-cart-total {
+  border-top: 1px solid var(--color-separator);
+  border-bottom: 1px solid var(--color-separator);
+}
+
+@media (--breakpoint-md) {
+  .commerce-cart-item {
+    border-bottom: 1px solid var(--color-separator);
+  }
+
+  .commerce-cart-button {
+    text-align: left;
+  }
+
+  .commerce-cart-item-image {
+    max-width: 100px;
+  }
+
+  .commerce-cart-summary {
+    height: calc(100vh - var(--wide-navbar-height));
```

Comment:
Ensure cart summary section on right side of wider devices is as tall as the page.